RELEASES.md

```diff
@@ -10,25 +10,39 @@
 
 - Direct usage of the following target properties has been deprecated. The names of the custom properties are
   no longer considered part of the public API and may change in the future. Instead, please use the functions
-  provided by corrosion.
+  provided by corrosion. Internally different property names are used depending on the CMake version.
   - `CORROSION_FEATURES`, `CORROSION_ALL_FEATURES`, `CORROSION_NO_DEFAULT_FEATURES`. Instead please use
     `corrosion_set_features()`. See the updated Readme for details.
   - `CORROSION_ENVIRONMENT_VARIABLES`. Please use `corrosion_set_env_vars()` instead.
   - `CORROSION_USE_HOST_BUILD`. Please use `corrosion_set_hostbuild()` instead.
+- The Minimum CMake version will likely be increased for the next major release. At the very least I want to drop 
```

Comment:
I think first person singular don't look so good in release note. Maybe it should rather be "we"

Reply:
Thanks for catching that!